src/ga4gh/vrs/utils/hgvs_tools.py

```diff
@@ -1,26 +1,143 @@
+import re
 import hgvs
+import hgvs.parser
+import hgvs.dataproviders.uta
+import hgvs.normalizer
+import hgvs.variantmapper
+import hgvs.sequencevariant
 
+from bioutils.accessions import coerce_namespace
+from ga4gh.vrs.dataproxy import _DataProxy
 
 class HgvsTools():
+    """
+    A convenience class that exposes only the tools needed by vrs-python for working with HGVS (Human Genome Variation Society) notation.
 
-    """ A utility class that provides access to hgvs components that have a UTA database dependency"""
-    def __init__(self, uta_conn):
+    Attributes:
+        parser (hgvs.parser.Parser): The HGVS parser object.
+        uta_conn: The UTA (Unified Transcriptome and Annotation) connection object.
```

Comment:
nitpick: it's "Universal Transcript Archive"

Reply:
fixed